### PR TITLE
UND-144 single select

### PR DESCRIPTION
**Pull request by Ashish-srij**
Single Select and Custom Select

### CODE REVIEW

File: stories/Molecules/CustomSelect/CustomSelect.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import './customselect.scss';
+import init from '../../assets/js/select-init';
+
+const CustomSelect = ({ text }) => {
+  useEffect(() => {
+    init();
+  }, []);
+  return (
+    <div>
```
Comment (surbhi90) [inline stories/Molecules/CustomSelect/CustomSelect.jsx:10]: @Ashish-srij Please remove this extra div

File: stories/assets/js/select.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        break;
+    }
+  }
+}
+// const customSelect = () => {
+//  const selects = $('[data-multi-select]');
+//  select.map((items, i) => {
+//
+//  });
+// };
```
Comment (surbhi90) [inline stories/assets/js/select.js:159]: @Ashish-srij Please remove this extra code

File: stories/Molecules/Select/select.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  text-transform: uppercase;
+}
+
+[dir='rtl'] {
+  .select-default {
+    background-position: left 22px center;
+    padding: 16px 16px 16px 50px;
+    text-align: right;
+  }
+}
```
Comment (surbhi90) [inline stories/Molecules/Select/select.scss:34]: @Ashish-srij Please use the proper variables and create the mixins

File: stories/Molecules/CustomSelect/customselect.scss [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+      text-align: right;
+
+      &::after {
+        left: 22px;
+        margin: 0;
+        right: auto;
+      }
+    }
+  }
+}
```
Comment (surbhi90) [inline stories/Molecules/CustomSelect/customselect.scss:111]: @Ashish-srij Please use the proper variables and create the mixins as we have similar styling the multiselect component also

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Molecules/Select/Select.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React, { useEffect } from 'react';
+import './select.scss';
+
+const Select = ({ text }) => (
+  <div>
```
Comment (surbhi90) [inline stories/Molecules/Select/Select.jsx:5]: @Ashish-srij Please remove extra div

**Review by surbhi90 — CHANGES_REQUESTED** [review]

File: stories/Molecules/CustomSelect/CustomSelect.jsx (hunk 1 above)
Comment (Ashish-srij) [reply stories/Molecules/CustomSelect/CustomSelect.jsx]: Done

File: stories/Molecules/Select/select.scss (hunk 3 above)
Comment (Ashish-srij) [reply stories/Molecules/Select/select.scss:29]: Done

File: stories/assets/js/select.js (hunk 2 above)
Comment (Ashish-srij) [reply stories/assets/js/select.js]: Done

File: stories/Molecules/CustomSelect/customselect.scss (hunk 4 above)
Comment (Ashish-srij) [reply stories/Molecules/CustomSelect/customselect.scss:107]: Done

File: stories/Molecules/Select/Select.jsx (hunk 5 above)
Comment (Ashish-srij) [reply stories/Molecules/Select/Select.jsx]: Done

File: stories/Molecules/CustomSelect/customselect.scss (hunk 4 above)
Comment (surbhi90) [reply stories/Molecules/CustomSelect/customselect.scss:107]: @Ashish-srij Still can see lot of repetitive code in this and multiselect so please create the mixins and placeholders

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**